README.md

```diff
@@ -52,7 +52,16 @@ yay -S tachidesk
 ```
 
 ### Docker
-Check this [github package](https://github.com/orgs/Suwayomi/packages/container/package/tachidesk) or our [dockerhub](https://hub.docker.com/repository/docker/suwayomi/tachidesk) for installation. Source code for containers is available at [this repository](https://github.com/Suwayomi/docker-tachidesk).
+Check our Offical Docker release [Tachidesk Container](https://github.com/orgs/Suwayomi/packages/container/package/tachidesk) or use [arbuilder's](https://github.com/arbuilder/Tachidesk-docker) tachidesk docker repo for installation. Source code for our container is available at [docker-tachidesk](https://github.com/Suwayomi/docker-tachidesk). By default the server will be running on http://localhost:4567 open this url in your browser.
```

Comment:
Isn't https://github.com/arbuilder/Tachidesk-docker the same as https://github.com/Suwayomi/docker-tachidesk ?
why link your repo?

Reply:
for dockerhub, dockerhub is annoying like for syncing Readme i have to use password in git action workflow Dockerhub PAT won't work. Also there is this https://www.docker.com/increase-rate-limits which is for pull request . I want to deprecate  https://hub.docker.com/r/arbuild/tachidesk-docker but until i can publish securely publish images to https://hub.docker.com/repository/docker/suwayomi/tachidesk without any potential issue it wont make sense. we can use github registry as our official release and add https://hub.docker.com/repository/docker/suwayomi/tachidesk later when some issues are resolved